transit/transit.go

```diff
@@ -38,7 +38,7 @@ type MetricKind string
 const (
 	MetricKindUnspecified MetricKind = "METRIC_KIND_UNSPECIFIED"
 	Gauge                            = "GAUGE"
-	Delta                            = "DELTA"
+	Delta                            = "Delta"
```

Comment:
Why change the case? Its inconsistent with GAUGE and CUMULATIVE. Also, any changes to transit.go must be checked with Data Geyser (C) code. Can we leave this as DELTA?

Reply:
Yes, we can compare strings ignoring the letter case if needed